source/meta.tex

```diff
@@ -1354,8 +1354,14 @@
 The set of scalar types for which this condition holds is
 \impldef{which scalar types have unique object representations}.
 \begin{note}
-If a type has padding bits, the condition does not hold;
-otherwise, the condition holds true for integral types.
+If a type \tcode{T} has padding bits, the condition does not hold for \tcode{T}.
+If \tcode{T} is an integral type other than \keyword{bool}, the condition
```

Comment:
This has lost the "otherwise": We can have integral types with padding bits, I think.

Reply:
With C++20 requiring two's complement, maybe not ... unless unsigned integers are padded too, and I think they can be.

Reply:
Yes, we can have a 31-bit unsigned int on CHAR_BIT=8 machine.